server/app.js

```diff
@@ -14,10 +14,11 @@ app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cookieParser());
 
+
 app.use(express.static(path.join(__dirname, '..', 'build')));
 
 app.get('/*', function (req, res) {
-  res.sendFile(path.join(__dirname, '..', 'build', 'index.html'));
```

Comment:
We need this piece of code so our node.js server app will be able to serve client side part.
Since you wanted to get some data in redux, you could just simple create a new endpoint, e.g. app.get('/events', () => {}) and change path from '/*' to '/' in this route if necessary.